src/99-appendices/08-coordinate-systems.md

```diff
@@ -136,4 +144,20 @@ presence MUST NOT produce a validation error.
 | fsaverage\[3&#124;4&#124;5&#124;6&#124;sym\]        | fsaverage\[&#124;Sym\] |
 | UNCInfant\[0&#124;1&#124;2\]V\[21&#124;22&#124;23\] | UNCInfant              |
 
+### Nonstandard coordinate system identifiers
+
+| Coordinate System | Description                                                                                                                                                                                                                                                           |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| individual        | Participant specific anatomical space (for example derived from T1w and/or T2w images). This coordinate system requires specifying an additional, participant-specific file to be fully defined. In context of surfaces this space has been refered to as `fsnative`. |
+| study             | Custom space defined using a group/study-specific template. This coordinate system requires specifying an additional file to be fully defined.                                                                                                                        |
```

Comment:
@effigies  and @robertoostenveld - perhaps those additional keys could be moved here and were originally created to reflect this scenario. i'm assuming `to be fully defined` means we have not defined it yet, rather than we know how to define it.

just a note that this study specific custom space could be surface-based or volume-based or some combination.

Reply:
@satra Could you make a specific suggestion? I'm not sure what you're proposing.

Reply:
how would a user define a custom or study-specific template/imagecoordinatesystem? 

`space-study` simply says the pipeline/dataset used a custom template but we still don't know what this is. so similar to the cifti use case, we need to know what this refers to. so something like this should be included in `spaces.json` as was originally discussed or some other place as relevant.

```
{
"ImageCoordinateSystem": "study",
"ImageCoordinateSystemReference": "/path/to/customtemplate"
"ImageCoordinateSystemUnits": "this key could be dropped similar to other named templates"
}
```

what if a study or pipeline used two or more custom templates? i know this is starting to go beyond common use cases, but should either be clarified that this is not supported and people should submit proposals or at least be considered in a section suggesting a study-specific imagecoordinatesystem.

Reply:
Isn't this covered by `SpatialReference`, with something like `space-study_bold.json` containing something like:

```
{
  "SpatialReference": "space-study_T1w.nii.gz"
}

Reply:
> Isn't this covered by SpatialReference, with something like space-study_bold.json

yes that would cover it. yes, then those keys can be dropped.

this would not cover the multiple study templates, so you may want to add a sentence to that effect.

Reply:
How about prefacing this table with:

```Markdown
The following coordinate system names are RECOMMENDED for individual- and study-specific reference spaces.
In order for these spaces to be interpretable, `SpatialReference` metadata MUST be provided, as described in
[Common file level metadata fields](05-derivatives/01-introduction.md#common-file-level-metadata-fields).

In the case of multiple study templates, additional names may need to be defined.
```

Reply:
sounds good for now. i wouldn't want this to hold up this PR.